integration-tests/src/test/resources/features/webhookaction.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Webhook Action tests
+
+  @webhookaction
+  Scenario: Webhook is correctly called
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+    And Webhook site request history is cleared
+
+    And add a Processor to the Bridge "mybridge" with body:
+    """
+    {
+      "name": "myProcessor",
+      "action": {
+        "type": "Webhook",
+        "parameters": {
+            "endpoint": "https://webhook.site/${system.webhook.site.uuid}"
+        },
+        "transformationTemplate" : "{\"text\": \"hello {data.name} by {id}\"}"
+      }
+    }
+    """
+    And the list of Processor instances of the Bridge "mybridge" is containing the Processor "myProcessor"
+    And the Processor "myProcessor" of the Bridge "mybridge" is existing with status "ready" within 3 minutes
+    And the Processor "myProcessor" of the Bridge "mybridge" has action of type "Webhook" and parameters:
+      | endpoint | https://webhook.site/${system.webhook.site.uuid} |
+
+    When send a cloud event to the Ingress of the Bridge "mybridge" with path "events":
+    """
+    {
+      "specversion": "1.0",
+      "type": "webhook.site.invoked",
+      "source": "WebhookActionTestService",
+      "id": "webhook-test",
+      "data": {
+          "name": "world"
+        }
+    }
+    """
+    Then the Ingress of the Bridge "mybridge" metric 'http_server_requests_seconds_count{method="POST",outcome="SUCCESS",status="200",uri="/events",}' count is at least 1
+    Then Webhook site contains request with text "hello world by ${bridge.mybridge.cloud-event.webhook-test.id}" within 1 minute
+
+
+    When delete the Processor "myProcessor" of the Bridge "mybridge"
+    
+    Then the Processor "myProcessor" of the Bridge "mybridge" is not existing within 2 minutes
+
+    And the Manager metric 'manager_processor_status_change_total{status="PROVISIONING",}' count is at least 1
+    And the Manager metric 'manager_processor_status_change_total{status="READY",}' count is at least 1
+    And the Manager metric 'manager_processor_status_change_total{status="DELETED",}' count is at least 1
```

Comment:
These metrics are already checked on https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/src/test/resources/features/processor.feature so I would remove it from here as they are unrelated to the WebHook test.

Reply:
@sutaakar sure, will do